src/Microsoft.Build.Sql/README.md

```diff
@@ -33,3 +35,36 @@ In Visual Studio 2019 (or newer), .NET Core targets can be used by default. Add
 <NetCoreBuild>True</NetCoreBuild>
 ```
 and the project can be built directly in VS 2019 or `dotnet build` from command line.
```

Comment:
instead of VS 2019, let's stick with "Visual Studio" since the version has already bumped to 2022